ci/rebuild_updated_recipes.py

```diff
@@ -63,15 +63,12 @@ def build(target_python, requirements):
     requirements = ','.join(requirements)
     logger.info('requirements: {}'.format(requirements))
     with current_directory('testapps/'):
-        try:
```

Comment:
I'm not sure why I had that try/except block in the first place if at then end it's to reraise without doing anything.
If we realise later on it turns out to be a smart thing, I'll put it back with a unit test or at least a comment next time :smile:

Reply:
Seems fine.